marketplace-service/src/main/java/com/axonivy/market/enums/NonStandardProduct.java

```diff
@@ -34,6 +35,8 @@ public enum NonStandardProduct {
   WORKFLOW_DEMO("workflow-demo", false, COMMON_IMAGES_FOLDER_NAME, "workflow/workflow-demos-product"),
   HTML_DIALOG_DEMO("html-dialog-demo", false, COMMON_IMAGES_FOLDER_NAME, "html-dialog/html-dialog-demos-product"),
   PROCESSING_VALVE_DEMO("processing-valve-demo", false, COMMON_IMAGES_FOLDER_NAME, ""),// no product json
+  DOC_FACTORY("doc-factory", false, "", "doc-factory-product"),
```

Comment:
I think you also add NonStandardProduct for aspose-barcode, because if the folder is sorted inordered. The issue is still happening